community/libmad/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+for patch in *.patch; do
+    patch -p1 < "$patch"
+done
+
+./configure \
+    prefix=/usr
```

Comment:
`--prefix=/usr`?

Reply:
either seems to work. but I'll change it :)

Reply:
fixed :)